packages/mdc-chips/chip/index.js

```diff
@@ -71,24 +73,12 @@ class MDCChip extends MDCComponent {
     this.id = this.root_.id;
     this.leadingIcon_ = this.root_.querySelector(strings.LEADING_ICON_SELECTOR);
     this.trailingIcon_ = this.root_.querySelector(strings.TRAILING_ICON_SELECTOR);
+    this.checkmark_ = this.root_.querySelector(strings.CHECKMARK_SELECTOR);
 
-    // Adjust ripple size for chips with animated growing width. This applies when filter chips without
-    // a leading icon are selected, and a leading checkmark will cause the chip width to expand.
-    const checkmarkEl = this.root_.querySelector(strings.CHECKMARK_SELECTOR);
-    if (checkmarkEl && !this.leadingIcon_) {
-      const adapter = Object.assign(MDCRipple.createAdapter(this), {
-        computeBoundingRect: () => {
-          const height = this.root_.getBoundingClientRect().height;
-          // The checkmark's width is initially set to 0, so use the checkmark's height as a proxy since the
-          // checkmark should always be square.
-          const width = this.root_.getBoundingClientRect().width + checkmarkEl.getBoundingClientRect().height;
-          return {height, width};
-        },
-      });
-      this.ripple_ = rippleFactory(this.root_, new MDCRippleFoundation(adapter));
-    } else {
-      this.ripple_ = rippleFactory(this.root_);
-    }
+    const adapter = Object.assign(MDCRipple.createAdapter(this), {
+      computeBoundingRect: () => this.foundation_.computeBoundingRect(),
```

Comment:
what about `this.foundation_.computeBoundingRectForRipple()`?

Reply:
Updated to getDimensions